docs/src/manifolds/power.md

```diff
@@ -1,5 +1,56 @@
 # Power manifold
 
+A power manifold is based on a [`Manifold`](@ref) $\mathcal M$ to build a
+$\mathcal M^{n_1 \times n_2 \times \cdots \times n_m}$.
+For the two special cases $m=1$ and $m=2$ we obtain
+one-dimensional data, i.e. vectors of points on $\mathcal M$ and matrices of images,
+respectively.
+
+## Example
+There are two ways to store the data: in a multidimensional array or in a nested array.
+
+Let's look at both for an example. Let $\mathcal M$ be `Sphere(2)` the 2-sphere. and we want
+to look at vectors of length 4.
+
+For the default, the [`ArrayPowerRepresentation`](@ref), we store the data in da multidimensional array,
+
+```@example 1
+using Manifolds #hide
+M = PowerManifold(Sphere(2),4)
+p = cat([1.0, 0.0, 0.0],
+        [1/sqrt(2.0), 1/sqrt(2.0), 0.0],
+        [1/sqrt(2.0), 0.0, 1/sqrt(2.0)],
+        [0.0, 1.0, 0.0]
+    ,dims=2)
```

Comment:
I would add here an example based on HybridArrays.jl. It's in the testset for `PowerManifold`, for example `HybridArray{Tuple{3,StaticArrays.Dynamic()}, Float64, 2}` for the case we have in this example.

Reply:
I added that to the example for completeness.

Reply:
OK, thanks!